pkg/gofr/gofr.go

```diff
@@ -339,10 +339,15 @@ func (a *App) PATCH(pattern string, handler Handler) {
 func (a *App) add(method, pattern string, h Handler) {
 	a.httpRegistered = true
 
+	reqTimeout, err := strconv.Atoi(a.Config.Get("REQUEST_TIMEOUT"))
```

Comment:
Why not using GetOrDefault?

Reply:
Not using `GetOrDefault` because in that case even if the doesn't want to set the request timeout we will force set it.

The idea is if user has set the config `REQUEST_TIMEOUT` then only we will set it , and if the user has given incorrect value for the config we will set it default to 5 seconds.

Reply:
Thanks it's clearer. Maybe the code could mention it in a comment